apps/src/lab2/views/Lab2ShareDialogWrapper.tsx

```diff
@@ -66,12 +69,24 @@ const Lab2ShareDialogWrapper: React.FunctionComponent<
   >(undefined);
 
   useEffect(() => {
-    if (channelId && projectType) {
-      getSubmissionStatus(channelId, projectType).then(response =>
-        setSubmissionStatus(response)
-      );
+    // Only signed-in users can submit projects to be considered for the featured project gallery.
+    if (channelId && projectType && isSignedIn) {
+      try {
+        getSubmissionStatus(channelId, projectType).then(response =>
+          setSubmissionStatus(response)
+        );
+      } catch (error) {
+        if (!(error instanceof NetworkError && error.response.status === 403)) {
+          MetricsReporter.logError({
```

Comment:
Just FYI that since this is the base metrics reporter, it will not automatically include the `AppName` dimension. We don't necessarily need to add it, just depends on if we want to split up these metrics by app name, or aggregate across all labs

Reply:
I think adding the `AppName` dimension would be handy. Will update!

Reply:
And just clarified with you offline that this will be for `incrementCounter` (not `logError`). Thanks!